module/VuFind/src/VuFind/Search/EDS/Options.php

```diff
@@ -304,7 +304,7 @@ protected function setCommonSettings($searchSettings, $setting, $list, $target)
         if (isset($searchSettings->General->$setting)) {
```

Comment:
Would make sense to make this a `!empty()` as well.

Reply:
Done.